### ISSUE TITLE
Set `aria-live` and `aria-busy` on outputs

### ISSUE THREAD

**Issue by jcheng5** [rstudio/shiny#2987]
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions
https://www.w3.org/TR/wai-aria-1.1/#aria-busy

**Comment by jooyoungseo** [comment]
selectize.js will be patched via #2993 to implement  `aria-live` to fetch dynamic contents. ionRangeSlider.js should be patched as that is completely inaccessible as of this writing.

**Comment by jooyoungseo** [comment]
Oh, I see. The point of this issue is that we have to implement `aria-live` for output region as well. That's great! We may want to address the input widget accessibility first, and will then be able to focus on the output panels.
